src/App.tsx

```diff
@@ -1,25 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [total, setTotal] = useState(items.length);
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const handlePageChange = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault();
+    const target = e.target as HTMLAnchorElement;
+    const value = target.getAttribute('href')?.substring(1);
+
+    setCurrentPage(prevPage => {
+      switch (value) {
+        case 'prev': {
+          return prevPage - 1;
```

Comment:
Ensure that `prevPage - 1` does not result in a page number less than 1. Consider adding a check to prevent this.